Add featured artist definition to Glossary (#2835)
ppy/osu-wiki: wiki/Glossary/en.md

Due to the [combo multiplier effect](#combo-multiplier-effect), full combos will give the most score in the osu!standard, osu!taiko, and osu!catch modes.

### Featured Artist

> A featured artist is someone who has officialy licensed music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping.

Reviewer: ```suggestion
*See also: [Featured Artists](https://osu.ppy.sh/beatmaps/artists)*

A featured artist is an artist who has officially licensed their music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping. All tracks from featured artists include a pre-timed `.osz` file to help you focus on beatmapping.
```

Reviewer 2: any reason for website link instead of wiki ?

Author: that should probably go into the description

Reviewer: @cl8n fwiw now, I thought there wasn't a wiki page for it

Due to the [combo multiplier effect](#combo-multiplier-effect), full combos will give the most score in the osu!standard, osu!taiko, and osu!catch modes.

### Featured Artist

> *See also: [Featured Artists](wiki/Featured_Artists)*

Reviewer: ```suggestion
*Main page: [Featured Artists](/wiki/Featured_Artists)*
```

> A featured artist is an artist who has officially licensed their music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping. All tracks from featured artists include a pre-timed `.osz` file to help you focus on beatmapping. You can find all of the artists and every person who is in the catalouge [here](https://osu.ppy.sh/beatmaps/artists)

Reviewer: ```suggestion
A featured artist is an artist who has officially licensed their music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping. All tracks from featured artists include a pre-timed `.osz` file to help you focus on beatmapping.
```
We should probably omit the last sentence since the main page would go into more detail.

Reviewer: ```suggestion
A featured artist is an artist who has officially licensed their music for use in osu!. Their music has been cleared by the osu! team and their songs are suitable for beatmapping.
```
or the last two
